packages/components/modal/tests/dialog.test.tsx

```diff
@@ -91,31 +91,33 @@ describe("<Dialog />", () => {
   })
 
   test("Dialog renders correctly when close", async () => {
-    const { user, findByTestId, getByText } = render(<DialogCloseExample />)
+    const { user, findByTestId, queryByTestId, getByText } = render(
+      <DialogCloseExample />,
+    )
 
     const openDialogButton = await findByTestId("OpenDialog")
 
     await user.click(openDialogButton)
     await expect(findByTestId("Dialog")).resolves.toBeInTheDocument()
     const cancelDialogButton = getByText("dialog-cancel")
     await user.click(cancelDialogButton)
-    waitFor(() => {
-      expect(screen.findByTestId("Dialog")).toBeNull()
```

Comment:
Why you removed `screen`?

Reply:
I'm sorry, but I'm returning it because there is no clear reason.